test/fixtures/links.txt

```diff
@@ -259,7 +261,8 @@ email@example.com
 (email@example.com)
 email@example.com
 
-
+(someone@example.com?subject=This%20is%20the%20subject&cc=someone_else@example.com&body=This%20is%20the%20body)
+someone@example.com?subject=This%20is%20the%20subject&cc=someone_else@example.com&body=This%20is%20the%20body
```

Comment:
It worth to add more test cases. For example, with dot after email and so on.

Note, that false positive is more painful than unrecognized pattern.